Task/services/users/functions.go

```diff
@@ -0,0 +1,34 @@
+package service
+
+import (
+	"regexp"
+
+)
+
+// validateId utility to validate the id
+func validateId(id interface{}) bool {
```

Comment:
we will not validate  here.

Reply:
well, I have createsd separate file for validation function which will be called from service.go